Tests/AL-Go-Helper.Test.ps1

```diff
@@ -271,114 +391,31 @@
         Remove-Item -Path $tempName -Recurse -Force
     }
 
-    It 'CheckAndCreateProjectFolder' {
-        Mock Write-Host { }
-
-        Push-Location
-
-        # Create a temp folder with the PTE template files
-        $tempName = Join-Path ([System.IO.Path]::GetTempPath()) ([Guid]::NewGuid().ToString())
-        New-Item $tempName -ItemType Directory | Out-Null
-        $repoName = "Per Tenant Extension"
-        $pteTemplateFiles = Join-Path $PSScriptRoot "../Templates/$repoName" -Resolve
-        Copy-Item -Path $pteTemplateFiles -Destination $tempName -Recurse -Force
-        $repoFolder = Join-Path $tempName $repoName
-        $repoFolder | Should -Exist
-        Join-Path $repoFolder '.AL-Go/settings.json' | Should -Exist
-        Set-Location $repoFolder
-
-        # Test without project name - should not change the repo
-        CheckAndCreateProjectFolder -project '.'
-        Join-Path $repoFolder '.AL-Go/settings.json' | Should -Exist
-        CheckAndCreateProjectFolder -project ''
-        Join-Path $repoFolder '.AL-Go/settings.json' | Should -Exist
-
-        # Create an app in an empty repo
-        New-Item -Path 'App' -ItemType Directory | Out-Null
-        Set-Content -Path 'App/app.json' -Value '{"id": "123"}'
-
-        # Creating a project in a single project repo with apps should fail
-        { CheckAndCreateProjectFolder -project 'project1' } | Should -Throw
-
-        # Remove app folder and try again
-        Remove-Item -Path 'App' -Recurse -Force
-
-        # Creating a project in a single project repo without apps should succeed
-        { CheckAndCreateProjectFolder -project 'project1' } | Should -Not -Throw
-
-        # .AL-Go folder should be moved to the project folder
-        Join-Path $repoFolder '.AL-Go/settings.json' | Should -Not -Exist
-        Join-Path '.' '.AL-Go/settings.json' | Should -Exist
-        'project1.code-workspace' | Should -Exist
-
-        # If repo is setup for multiple projects, using an empty project name should fail
-        Set-Location $repoFolder
-        { CheckAndCreateProjectFolder -project '' } | Should -Throw
-
-        # Creating a second project should not fail
-        { CheckAndCreateProjectFolder -project 'project2' } | Should -Not -Throw
-        Join-Path $repoFolder 'project2/.AL-Go/settings.json' | Should -Exist
-        Join-Path $repoFolder 'project2/project2.code-workspace' | Should -Exist
-
-        # Clean up
-        Pop-Location
-        Remove-Item -Path $tempName -Recurse -Force
+    It 'Settings schema is valid' {
+        Test-Json -json $schema | Should -Be $true
     }
 
-    It 'GetFoldersFromAllProjects' {
-        Mock Write-Host { }
-        Mock Out-Host { }
+    It 'Default settings match schema' {

```

Comment:
Can we also check that all default settings are present within the syntax file? Just so we don't forget to add new settings to the syntax file